msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionTopicBean.java

```diff
@@ -84,6 +84,8 @@
   private String locked = "";
   private String moderated = "";
   private String postFirst = "";
+  private String postAnonymous = "";
```

Comment:
I was just being consistent. It's possible something we're doing in the JSP files requires them to be Strings, although I think it's unlikely